runtime/tutor

```diff
@@ -1186,6 +1186,160 @@ multiple cursors, they won't be uncommented but commented again.
 
 
 
```

Comment:
I think the first section should begin with `CHAPTER 12 MATCH MODE`. It seems to begin on 12.1, maybe this was an oversight?